src/borg/testsuite/archiver/debug_cmds.py

```diff
@@ -63,6 +64,64 @@ def test_debug_put_get_delete_obj(self):
         output = self.cmd(f"--repo={self.repository_location}", "debug", "delete-obj", "invalid")
         assert "is invalid" in output
 
+    def test_debug_id_hash_format_put_get_parse_obj(self):
+        """Test format-obj and parse-obj commands"""
+
+        self.cmd(f"--repo={self.repository_location}", "rcreate", RK_ENCRYPTION)
+        data = b"some data" * 100
+        meta_dict = {"some": "property"}
+        meta = json.dumps(meta_dict, ensure_ascii=False).encode()
```

Comment:
do you need to give `ensure_ascii=False`?

Reply:
Yes, https://stackoverflow.com/questions/18337407/saving-utf-8-texts-with-json-dumps-as-utf-8-not-as-a-u-escape-sequence